public/schema.json

```diff
@@ -46,54 +46,56 @@
         },
         "required" : [ "enrolment" ]
       },
-      "uk.gov.hmrc.securemessage.controllers.model.cdcm.Recipient" : {
+      "uk.gov.hmrc.securemessage.controllers.models.generic.Recipient" : {
         "properties" : {
           "customer" : {
-            "$ref" : "#/components/schemas/uk.gov.hmrc.securemessage.controllers.model.cdcm.Customer"
+            "$ref" : "#/components/schemas/uk.gov.hmrc.securemessage.controllers.models.generic.Customer"
           }
         },
         "required" : [ "customer" ]
       },
-      "uk.gov.hmrc.securemessage.controllers.model.cdcm.System" : {
+      "uk.gov.hmrc.securemessage.controllers.models.generic.System" : {
         "properties" : {
           "name" : {
             "type" : "string"
           },
+          "display" : {
+            "type" : "string"
+          },
           "parameters" : {
+            "nullable" : true,
             "type" : "object",
             "additionalProperties" : {
               "type" : "string"
             }
-          },
-          "display" : {
-            "type" : "string"
           }
         },
-        "required" : [ "name", "parameters", "display" ]
+        "required" : [ "name", "display" ]
       },
-      "uk.gov.hmrc.securemessage.controllers.model.cdcm.Sender" : {
+      "uk.gov.hmrc.securemessage.controllers.models.generic.Sender" : {
         "properties" : {
           "system" : {
-            "$ref" : "#/components/schemas/uk.gov.hmrc.securemessage.controllers.model.cdcm.System"
+            "$ref" : "#/components/schemas/uk.gov.hmrc.securemessage.controllers.models.generic.System"
           }
         },
         "required" : [ "system" ]
       },
-      "uk.gov.hmrc.securemessage.controllers.model.cdcm.ConversationRequest" : {
+      "uk.gov.hmrc.securemessage.controllers.models.generic.ConversationRequest" : {
         "properties" : {
           "sender" : {
-            "$ref" : "#/components/schemas/uk.gov.hmrc.securemessage.controllers.model.cdcm.Sender"
+            "$ref" : "#/components/schemas/uk.gov.hmrc.securemessage.controllers.models.generic.Sender"
           },
           "recipients" : {
             "type" : "array",
             "items" : {
-              "$ref" : "#/components/schemas/uk.gov.hmrc.securemessage.controllers.model.cdcm.Recipient"
+              "$ref" : "#/components/schemas/uk.gov.hmrc.securemessage.controllers.models.generic.Recipient"
```

Comment:
from model.cdcm to model.generic, is that intended? I know Matt had that in reverse

Reply:
Yes it is intended. Oleg specifically requested that we change this as it is no longer CDCM specific following discussions earlier today.